README.md

```diff
@@ -235,6 +235,34 @@ UniversalBle.enableBluetooth();
 UniversalBle.disableBluetooth();
 ```
 
+### Request MTU
+
+This method will **attempt** to set the MTU (Maximum Transmission Unit) but it is not guaranteed to succeed due to platform limitations. It will always return the current MTU.
+
+```dart
+int mtu = await UniversalBle.requestMtu(widget.deviceId, 247);
+```
+
+#### Platform Limitations
+
+On most platforms, the MTU can only be queried but not manually set:
+
+- **iOS/macOS**: System automatically sets MTU to 185 bytes maximum
+- **Android 14+**: System automatically sets MTU to 517 bytes for the first GATT client
+- **Windows**: MTU can only be queried
+- **Linux**: 
```

Comment:
@rohitsangwan01 not sure about the situation on Linux.

Reply:
am also not sure if we can do this in linux, cant find such option in Bluez plugin, but in our current implementation, Mtu can be queried only, but cant be set